scripts/e2e/bootstrap_job/Makefile

```diff
@@ -6,12 +6,15 @@ REGISTRY ?=gcr.io/cluster-api-provider-vsphere/pr/ci
 all: build
 .PHONY : all
 
+genspec:
+	sh ../generate-e2e-spec.sh
```

Comment:
Don't call this with a `sh` in front of it. the script already has a shebang for it, e.g. `#!/bin/bash`. So this is calling the script as shell, even though it is in bash.